tests/python/relax/test_transform_gradient_checkpoint.py

```diff
@@ -16,12 +16,13 @@
 # under the License.
 """Unit tests for gradient with checkpointing."""
 import tvm
-from tvm.relax.block_builder import BlockBuilder
```

Comment:
These changes are only style enhancement (reordering imports, deleting redundant "dtype=" annotation). I only add two tests for it (test_checkpoint_dag and test_checkpoint_with_intermediate_require_grads). See the end of this file.